src/Mvc/test/WebSites/BasicWebSite/Controllers/AsyncActionsController.cs

```diff
@@ -26,6 +26,17 @@ public override void OnActionExecuted(ActionExecutedContext context)
             }
         }
 
+        public async Task<IActionResult> ActionWithSuffixAsync()
+        {
+            await Task.Yield();
+            return Ok();
+        }
+
+        public IActionResult ActionReturningViewAsync()
```

Comment:
Very minor but it looks strange to have a sync method named with an Async suffix.